rotkehlchen/globaldb/handler.py

```diff
@@ -1011,6 +1011,108 @@ def add_historical_prices(entries: List['HistoricalPrice']) -> None:
 
         connection.commit()
 
+    @staticmethod
+    def add_single_historical_price(entry: HistoricalPrice) -> None:
```

Comment:
This function should also return some information. There is no reason to return `None`. The least you can do is returned if the price was written in the DB or not.